contrib/http_util/src/main/java/io/opencensus/contrib/http/util/HttpMeasureConstants.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.http.util;
+
+import io.opencensus.stats.Measure;
+import io.opencensus.stats.Measure.MeasureDouble;
+import io.opencensus.stats.Measure.MeasureLong;
+import io.opencensus.tags.TagKey;
+
+/**
+ * A helper class which holds OpenCensus's default HTTP {@link Measure}s and {@link TagKey}s.
+ *
+ * <p>{@link Measure}s and {@link TagKey}s in this class are all public for other
+ * libraries/frameworks to reference and use.
+ *
+ * @since 0.13
+ */
+public class HttpMeasureConstants {
+
+  private HttpMeasureConstants() {}
+
+  private static final String UNIT_COUNT = "1";
+  private static final String UNIT_SIZE = "By";
+  private static final String UNIT_LATENCY = "ms";
+
+  /**
+   * {@link Measure} for the number of client-side HTTP requests started.
+   *
+   * @since 0.13
+   */
+  public static final MeasureLong HTTP_CLIENT_REQUEST_COUNT =
+      Measure.MeasureLong.create(
+          "opencensus.io/http/client/request_count",
+          "Number of client-side HTTP requests started",
+          UNIT_COUNT);
+
+  /**
+   * {@link Measure} for the client-side HTTP request body size if set as ContentLength
+   * (uncompressed).
+   *
+   * @since 0.13
+   */
+  public static final MeasureLong HTTP_CLIENT_REQUEST_BYTES =
+      Measure.MeasureLong.create(
+          "opencensus.io/http/client/request_bytes",
+          "Client-side HTTP request body size if set as ContentLength (uncompressed)",
+          UNIT_SIZE);
```

Comment:
Should we specify that the size is uncompressed in the measure and constant names?  I'm not suggesting making a change in this PR, though, since it's from opencensus-specs.